styleq.flow.js

```diff
@@ -8,11 +8,12 @@
  */
 
 export type CompiledStyle = {
-  $$css: boolean,
+  $$css: true,
   [key: string]: string,
 };
 
 export type InlineStyle = {
+  $$css?: empty,
```

Comment:
Is this how you type a property that "must not be included"?

Reply:
Yes. Defining this key explicitly is important to create a "disjoint union" so that the two branches for "CompiledObject" and "InlineStyle" know what kind of object they're each dealing with.